contracts/tgrade-valset/src/msg.rs

```diff
@@ -395,25 +395,38 @@ impl OperatorResponse {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct JailingPeriod {
+    pub start: Timestamp,
+    pub end: JailingEnd,
+}
```

Comment:
I can't help but notice that these could be a pair of `u64` values, with the convention that `u64::MAX` (or `0`) means forever.

But, who am I to judge? :-)